client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,7 +113,7 @@ const CartItem = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else if ( isTheme( this.props.cartItem ) ) {
-			info = this.props.selectedSite.domain;
+			info = this.props.selectedSite && this.props.selectedSite.domain;
```

Comment:
My bad. I thought `meta` is not used for themes, but it seems to be used. 
![theme meta](https://cloud.githubusercontent.com/assets/1103398/23753185/c4b8b9e4-04d8-11e7-8f96-0092c83a8fab.png)